sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/manager/DownloadManagerRequestTask.java

```diff
@@ -35,6 +37,7 @@ protected Void doInBackground(Void... params) {
         AppCenterLog.debug(LOG_TAG, "Start downloading new release from " + downloadUrl);
         DownloadManager downloadManager = mDownloader.getDownloadManager();
         DownloadManager.Request request = createRequest(downloadUrl);
+        request.setTitle(String.format("%s %s (%s)", mTitle, releaseDetails.getShortVersion(), releaseDetails.getVersion()));
```

Comment:
You cacn put `%s (%s)` directly to the string (localized). It's required because ordering can be different in some languages